web_development_101/the_basics/introduction_to_web_development.md

```diff
@@ -53,43 +52,43 @@ These are some of the basic tools you will use regularly. You may not know what
 
 ### Motivation
 
-Learning to code is incredibly rewarding but it can also be difficult and frustrating.
+Learning to code is incredibly rewarding but can also be difficult and frustrating.
 The strongest assets you can have as a student are a desire to build, a problem-solving mind, and persistence in the face of setbacks.
 
 The web development industry has a long history of successful developers with varying backgrounds, so people tend to care more about what you've actually **built** than how you got there.
 
-So you need to build. Build and build and build and keep on building as much as you can. Build projects that try out new technologies and build things that you need.
-Once you start to get comfortable, start hacking on open source projects [(like The Odin project itself)](/contributing). The more you practice, the more you will learn about what you can do and the closer you will get to being hireable.
+So, you need to build. Build and build and build and keep on building as much as you can. Build projects to try out new technologies, and build things that you need.
+Once you start to get comfortable with the tools, start hacking on open source projects [(like The Odin project itself)](/contributing). The more you practice, the more you will learn about what you can do, and the closer you will get to being hireable.
 
 Read [this comprehensive blog post from Happy Bear Software](https://www.happybearsoftware.com/how-to-get-a-programmer-job) about the journey to getting hired as a brief introduction to what you will face ahead.
 
 ### Why Odin?
 
 I want you to know that this will not be easy.
 
-There are plenty of other online curriculums for beginners, but they are often taught in an extremely isolated and controlled environment, and only cover a specific topic.
+There are plenty of other online curriculums for beginners, but they are often taught in an extremely isolated and controlled environment and cover only a specific topic.
 
-The Odin Project takes a realistic view of what you need to know, and has you set up and work in your own environment, much like what you'll be doing when you get a job.
+The Odin Project takes a realistic view of what you need to know and has you set up and work in your own environment, much like what you'll be doing when you get a job.
 
 It acknowledges that you need a wide variety of skillsets and languages to reach an employable level.
 
-It is constantly evolving because of people like you who further along will oftentimes pay it forward and incrementally improve our content again and again.
+The Odin Project is constantly evolving because of people like you who get further along in the curriculum and pay it forward by incrementally improving our content over time.
```

Comment:
Because of people like you @leila-alderman 😄

Reply:
Thanks! Happy to help out!